app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
```

Comment:
I think Hyrax use the mailboxer gem but adding the base class as an option to allow us to extend it for other mailing services that we may need. We can just remove it now and add it in later when we need other/different mailing services. Do you like to remove it for now?